Tasks/Common/find-files-legacy/npm-shrinkwrap.json

```diff
@@ -1,113 +1,162 @@
 {
   "name": "find-files-legacy",
   "version": "1.0.0",
+  "lockfileVersion": 1,
```

Comment:
>lockfileVersion": 1 [](start = 3, length = 19)

build in 142 and see if this gets regenerated

Reply:
reverted this. don't think it would have caused a problem though